### ISSUE TITLE
Exposed credentials?

### ISSUE THREAD

**Issue by simonhaenisch** [Logflare/next-pino-logflare-logging-example#4]
Hey, you're setting the credentials here:

https://github.com/Logflare/next-pino-logflare-logging-example/blob/0cfd2b012e0a0eea7138358ce99fcd7befcc708f/logger/logger.js#L8-L11

but then you're also using the logger for client-side logging here:

https://github.com/Logflare/next-pino-logflare-logging-example/blob/0cfd2b012e0a0eea7138358ce99fcd7befcc708f/pages/index.js#L9

Doesn't that mean that the API key would be exposed in the frontend source code? Maybe that wouldn't be so much of an issue if it was possible to limit ingestion to come from a specific set of domains, but I couldn't find any setting for that either?

**Comment by ImLunaHey** [comment]
Did you ever get an update on this?

**Comment by simonhaenisch** [comment]
No, I stopped using Logflare a while ago (this wasn't the reason though cause I was only using it for backend logging anyway). Eventually I switched to a product which allows better visualizations from logs (and has a better pricing strategy).

**Comment by goustaff** [comment]
Ping @chasers 
Would be glad to hear from you about this issue. I'm also uncomfortable with it.
